cloudapp/src/app/main/main.component.ts

```diff
@@ -14,13 +14,15 @@ import { catchError, concatMap, map, tap, toArray} from "rxjs/operators";
 export class MainComponent implements OnInit, OnDestroy{
 
     numRecordsToPrint: number = 0;
+    pageLoaded: boolean = false;
     private currentUserActions;
     private pageLoadSubscription:Subscription;
     private pageLoadedSubject = new Subject<Entity[]>();
 
     pageLoadedAction$ = this.pageLoadedSubject.asObservable().pipe(
         concatMap(entities => this.userService.users$(entities)),
         tap(currentUserAction => this.currentUserActions = currentUserAction),
+        tap(() => this.pageLoaded=true),
```

Comment:
Seems to be missing some spacing;) Normally my eye doesn't catch these small things but the rest of the code uses spacing consequently except this and line 56.

Reply:
👍

Reply:
@jorntx thanks for the input! I fixed and pushed the changes :-)